src/data/markdown/translated-guides/en/03 Results output/200 Real-time/00 Prometheus remote write.md

```diff
@@ -44,7 +44,7 @@ you can use the `K6_PROMETHEUS_RW_TREND_AS_NATIVE_HISTOGRAM=true` environment va
 (or one of the other ways).
 The output then converts all the trend types into a dedicated Native Histogram.
 
-Mapping trends as native histograms gives you more efficient storage and preciser queries.
+Mapping trends as native histograms gives you more efficient storage and precise queries.
```

Comment:
```suggestion
Mapping trends as native histograms gives you more efficient storage and more precise queries.
```
I still think we should use the comparative for both items, otherwise it could be ambiguous.